client/app/dedicated/server/dedicated-server.controller.js

```diff
@@ -226,6 +227,14 @@ angular.module('App').controller('ServerCtrl', (
       $scope.urlRenew = url;
     });
 
+    Server.getUsbStorageInformations($stateParams.productId).then(
+      (result) => {
+        if (_.isArray(result) && result[1].usbKeys) {
```

Comment:
```suggestion
        const [, { usbKeys }] = result;
        if (usbKeys) {
```

Reply:
Since this ticket has been validated by QA team I will merge it.

Suggested change could be applied in a separate pull request.

Thanks!